### PR TITLE
feature (ref 36588): set a permission to give access to users base on…

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Ticket:** https://yaits.demos-deutschland.de/T36588

Description: Institution_Sachbearbeitung and Fachplaner-Sachbearbeiter (PLANNING_AGENCY_WORKER and PUBLIC_AGENCY_WORKER) should not have a save/cancel option in my data form, because of that I created a new permission and use that in every project

### How to review/test
go to mein Daten and check these roles have those options or not

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: config/permissions.yml [hunk 1]
```diff
@@ -1622,6 +1622,10 @@ feature_project_switcher:
     label: 'Display a dropdown menu in global header to allow users to quickly move to other instances of the SH platform.'
     loginRequired: false
     parent: area_admin
+feature_profile_editable:
+    description: 'If enabled the user can his/her data edit'
```
Comment (mussbach) [inline config/permissions.yml:1626]:
```suggestion
    description: 'If enabled the user can edit his/her data'
```

**Review by mussbach — CHANGES_REQUESTED** [review]

File: config/permissions.yml (hunk 1 above)
Comment (jalalifardemos) [reply config/permissions.yml]: done in [4e6dcac](https://github.com/demos-europe/demosplan-core/pull/2800/commits/4e6dcac3763f01ece3fe13722f0de9c8b68bc519)

**Review by MoritzMandler — APPROVED** [review]
lgtm

**Comment by jalalifardemos** [comment]
https://github.com/demos-europe/demosplan-core/pull/2812

**Comment by jalalifardemos** [comment]
https://github.com/demos-europe/demosplan-core/pull/2812

**Pull request closed without merging**